parse_pymon.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
Maybe move this to the tests directory, so as not to clutter the main directory too much? And please put code that runs stuff in a function and the add a line
```python
if __name__ == '__main__':
    run_my_function()
```
so just importing the module does not execute it, because that will lead to problems.

Reply:
Or put it in the github/workflows directory if it's only relevant for this workflow?

Reply:
ah right, that's true, I added it to the ignore list :grin:

Reply:
well, we could use it manually at any point, outside the GA dir, so prob best in `tests`